src/plugins/intel_gpu/src/graph/shape_of.cpp

```diff
@@ -12,36 +12,33 @@
 namespace cldnn {
 GPU_DEFINE_PRIMITIVE_TYPE_ID(shape_of)
 
-layout shape_of_inst::calc_output_layout(shape_of_node const& node, kernel_impl_params const& impl_param) {
-    auto prim = impl_param.typed_desc<shape_of>();
-
-    data_types dt = data_types::i32;
-
-    if (prim->output_data_types[0])
-        dt = *prim->output_data_types[0];
-
+namespace {
+data_types get_output_data_type(const kernel_impl_params& impl_param) {
```

Comment:
please change this function to static if it is only needed for a local usage.

Reply:
This function is nested into anonymous namespace, so it is already static.